CHANGELOG-0.10.md

```diff
@@ -0,0 +1,183 @@
+Heptio Ark v0.10 contains a number of breaking changes.  Upgrading will require some additional steps beyond just updating your client binary and your
+container image tag. We've provided a [detailed set of instructions][1] to help you with the upgrade process. **Please read and follow these instructions 
```

Comment:
This is good; I was thinking of a separate section with like a `### Breaking Changes` header to call it out very loudly. IMO there's no such thing as overkill in communicating this, even if some people will miss it.

Reply:
As an addition to the breaking changes, I think it'd be worth highlighting what plugin authors should know to upgrade.

Things like:

* new `prefix` parameter on `ListCommonPrefixes`
* The new `NewServer` function and the associated `Register` functions.

This can be short, but I think those hints would be helpful.

Reply:
@nrb could you write the plugin change notes, and I'll add it?

Reply:
pushed an update - let me know what you think

Reply:
@skriss Sure thing, will do that now.